software/base/src/main/java/brooklyn/entity/brooklynnode/BrooklynEntityMirror.java

```diff
@@ -54,6 +56,9 @@
         "Frequency to poll for client sensors", Duration.FIVE_SECONDS);
     
     public static final AttributeSensor<String> MIRROR_STATUS = Sensors.newStringSensor("brooklyn.mirror.monitoring_status");
+    @SuppressWarnings("rawtypes")
+    public static final AttributeSensor<Map> MIRROR_SUMMARY = Sensors.newSensor(Map.class, "brooklyn.mirror.summary");
```

Comment:
add description, to say that this is the json map returned by the entity rest endpoint (ie the `EntitySummary` REST model)